tensorflow/python/util/compat.py

```diff
@@ -138,10 +141,10 @@ def path_to_str(path):
 
   Examples:
   ```python3
-  >>> tf.compat.path_to_str('C:\\XYZ\\tensorflow\\./.././tensorflow')
-  'C:\\XYZ\\tensorflow\\./.././tensorflow' # Windows OS
+  >>> tf.compat.path_to_str('C:\XYZ\tensorflow\./.././tensorflow')
```

Comment:
I don't think you should remove `\`. `\\t` is not the same as `\t` (the first one is a two character string containing a backslash and a t whereas the second one is a single character string containing the tab character.